App.js

```diff
@@ -1,10 +1,21 @@
 import React, { Component } from 'react';
+import { createStackNavigator } from 'react-navigation';
 
-import Todo from './src/app/screens/Todo';
+import TabNav from './src/app/screens/TabNav';
+import { ROUTES } from './src/constants/routes';
+
+const RootStack = createStackNavigator(
+  {
+    navbar: TabNav
+  },
+  {
+    initialRouteName: ROUTES.navbar
+  }
+);
```

Comment:
move this to `app/screens.js` and import it here

Reply:
also rename `TabNav` to `Home`

```
export default createStackNavigator({
  [Routes.Home]: {
    screen: Home
  },
});
```

Reply:
Also, instead of defining the navigationOptions in each screen, define them here

```
export default createStackNavigator({
  [Routes.Home]: {
    screen: Home,
    navigationOptions: {
       title: screenTitles.HOME
    }
  }
});
```

Reply:
Also, if you want to, you could define the tab navigator here,,,


```
[Routes.Home]: {
      screen: TabNavigator(
        {
          [Routes.Todo]: {
            screen: Todo,
            navigationOptions: (...)
          },
          [Routes.Book]: {
            screen: Book,
            navigationOptions: (...)
          }
        },
       <<Optional: you could send navigation options for the tab container...>>
      ),
       <<Optional: you could send navigation options for the 'HOME' screen ...>>
    },
```